util/devel/nextLinter

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python3
```

Comment:
This file could use some more comments. Also I wonder if we should put it in compiler/next/util?